### DISCUSSION TITLE
too many console error

### DISCUSSION THREAD

**Discussion by hakanozkaptan** [ritikbanger/react18-input-otp#4] [Q&A] [answered]
I think also need to delete this control
```
      if (placeholder.length > 0) {
        console.error('Length of the placeholder should be equal to the number of inputs.');
      }
```

to much console.error [image](https://pasteboard.co/BKw35swxmVsP.png)

**Comment by ritikbanger** [comment, marked as answer]
There is nothing wrong with this. It is only an indicator that the placeholder you have given does not equal to the numInput prop.

**Reply by ritikbanger** [reply]
@hakanozkaptan Let me know if you have any questions over the same or can I close this discussion?

**Comment by hakanozkaptan** [comment]
I think this kind of warning should be made by the developer on the client side. It seems to me that the 3rd party plugin does not need to give such a warning.

**Reply by ritikbanger** [reply]
```
  getPlaceholderValue = () => {
    const { placeholder, numInputs } = this.props;

    if (typeof placeholder === 'string') {
      if (placeholder.length === numInputs) {
        return placeholder;
      }

      if (placeholder.length > 0) {
        console.error('Length of the placeholder should be equal to the number of inputs.');
      }
    }
  };
```

Here, we are returning the placeholder given by user if the length of the placeholder is equal to numInputs but we also have to consider the case where placeholder has some length which is not equal to the numInputs that's why we are giving this error. If you have any suggestion other than this to handle this case, please suggest.

**Reply by ritikbanger** [reply]
@hakanozkaptan Do you have any more query or should I close this discussion?

**Reply by hakanozkaptan** [reply]
You can close thank you
